### PR TITLE
Fix for LLDP portname issue

### PR DESCRIPTION
**Pull request by sumukhatv**
First check for operstate and if its not present then check for ifindex

**- What I did**
Fixed the issue in LLDP where portname was advertised as a MAC address instead
**- How I did it**
To decide whether a port exists, first we check for operstate file. If this file is present, then we check if the state is "up" or not. If the file itself is not present, then we know that it is a v-switch, so we can now look for ifindex file and make a decision whether the port exists or not.

**- How to verify it**

**- Description for the changelog**

To decide the existence of a port, check for status in the operstate file. If this file is not present check ifindex file.  

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: dockers/docker-lldp-sv2/lldpmgrd [hunk 1]
```diff
@@ -74,11 +74,20 @@ def signal_handler(sig, frame):
...
-
-    return True
+    filename = "/sys/class/net/%s/operstate" % port_name
+    if os.path.exists(filename):
+        with open(filename) as fp:
+            state = fp.read()
+            if "up" in state:
+                return True
+            else:
+                return False
```
Comment (pavel-shirshov) [inline dockers/docker-lldp-sv2/lldpmgrd]: We can write it shorter
```return "up" in state```

File: dockers/docker-lldp-sv2/lldpmgrd [hunk 2]
```diff
@@ -74,11 +74,20 @@ def signal_handler(sig, frame):
...
+    else:
+        filename = "/sys/class/net/%s/ifindex" % port_name
+        if os.path.exists(filename):
+            return True
+        else:
+            return False
```
Comment (pavel-shirshov) [inline dockers/docker-lldp-sv2/lldpmgrd]: we can write it shorter
```return os.path.exists(filename)```

File: dockers/docker-lldp-sv2/lldpmgrd (hunk 1 above)
Comment (sumukhatv) [reply dockers/docker-lldp-sv2/lldpmgrd]: Will do this. Thanks

File: dockers/docker-lldp-sv2/lldpmgrd (hunk 2 above)
Comment (sumukhatv) [reply dockers/docker-lldp-sv2/lldpmgrd]: Sure. I will change this too.

**Pull request merged**